src/Reflex/Dom/Builder/Static.hs

```diff
@@ -178,7 +178,7 @@ instance SupportsStaticDomBuilder t m => DomBuilder t (StaticDomBuilderT t m) wh
     --TODO: Do not escape quotation marks; see https://stackoverflow.com/questions/25612166/what-characters-must-be-escaped-in-html-5
     shouldEscape <- asks _staticDomBuilderEnv_shouldEscape
     let escape = if shouldEscape then fromHtmlEscapedText else byteString . encodeUtf8
-    modify . (:) <=< hold (escape initialContents) $ fmap escape setContents
+    modify . (:) <=< hold (escape initialContents) $ fmapCheap escape setContents --Only because it doesn't get optimized when profiling is on
```

Comment:
Let's clarify this comment?  It's clear *now* what is "only because", but it won't be to future readers.